cves/kernel/CVE-2016-4998.yml

```diff
@@ -467,11 +420,10 @@ CWE_instructions: |
 CWE:
 - 119
 CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+  manually confirmed
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
-CVSS:
+nickname: out of blob memory access
```

Comment:
Looking at the other vulnerabilities on the VHP website, this should be capitalized like a title for consistency.